assets/src/components/reporting/ReportingDashboard.tsx

```diff
@@ -183,13 +205,23 @@ class ReportingDashboard extends React.Component<Props, State> {
     );
   };
 
+  formatResponseTimeStats = (responseTime: number) => {
+    const seconds = Math.round(responseTime);
+    //time would look like 00:01:20 if on average it takes 80 seconds to respond
+    const time = new Date(seconds * 1000).toISOString().substr(11, 8);
```

Comment:
i think `substr` is a legacy function (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/substring#The_difference_between_substring_and_substr)

but we can just do
```js
new Date(seconds * 1000).toISOString().substring(11, 19);
```

also since this is a pure function that doesn't require any component state and seems like it could be useful in other cases outside of the context of this component, we should probably just move it out into `/reporting/support.ts` so we can test it more easily (once we have more frontend tests 😛) -- what do you think?

Reply:
Yeah sounds good I'll move it out